hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/DBStoreBuilder.java

```diff
@@ -298,6 +308,18 @@ private ManagedColumnFamilyOptions getDefaultCfOptions() {
         .orElseGet(defaultCfProfile::getColumnFamilyOptions);
   }
 
+  /**
+   * Get default column family options, but with column family write buffer
+   * size limit overridden.
+   * @param writeBufferSize Specify column family write buffer size.
+   * @return ManagedColumnFamilyOptions
+   */
+  private ManagedColumnFamilyOptions getDefaultCfOptions(long writeBufferSize) {
```

Comment:
I'd prefer delaying any further refactoring of a shared class like `DBStoreBuilder` until we merge to master to prevent headaches around merge conflicts. Would you like to file a jira for this so we can fix this after the merge?

It is still the "default" in the sense that we are still getting all the other default CF options. Just with write buffer size explicitly overridden, as mentioned in the javadoc. Similar to `getDefaultDBOptions(tableConfigs)` right below.

Reply:
Renamed to `getCfOptions`